src/externals/mct/mct/m_GlobalSegMap.F90

```diff
@@ -1764,22 +1761,25 @@ subroutine active_pes_(GSMap, n_active, pe_list)
   character(len=*),parameter :: myname_=myname//'::active_pes_'
 
   integer :: count, i, n, ngseg, ierr
-  logical :: new
-  integer, dimension(:), allocatable :: temp_list
-  integer, dimension(:), allocatable :: perm
+  integer :: max_activepe, p
```

Comment:
Did you bring in some changes from @jedwards4b in that branch?  It was definitely one of you two.   ESMCI's master version is different from the current master in mpi-serial or MCT.  

Anyway we don't need to figure out "who" because the "why" appears to just be formatting.